packages/css/js/Css_Js_Core.ml

```diff
@@ -1482,13 +1482,6 @@ let zoomIn = Cursor.zoomIn
 let zoomOut = Cursor.zoomOut
 let vw x = `vw x
 let fr x = `fr x
-
-module Calc = struct
-  let ( - ) a b = `calc (`sub (a, b))
-  let ( + ) a b = `calc (`add (a, b))
-  let ( * ) a b = `calc (`mult (a, b))
-end
-
```

Comment:
Removed because it was redundant

Reply:
It's not, please revert

Reply:
I don't understand how it is used.

Reply:
It's part of the public API, so some users might have it on their code